SUPPORT.md

```diff
@@ -0,0 +1,49 @@
+# Support and Compatibility for fabric-chaincode-node
+
+Github is used for code base management, issues should reported in the [FABCN](https://jira.hyperledger.org/projects/FABCN/issues/) component in JIRA.
+
+
+## Summary of Compatibility
+
+Summary of the Fabric versions and the default node runtimes they use to host chaincode and contracts; along side the version of the node modules and which runtimes they are compatible with.
+
+|                       | Fabric 1.4 (node v8) | Fabric v2.0 (node v12) |
+| --------------------- | -------------------- | ---------------------- |
+| node modules **v1.4** | Yes                  | Yes                    |
```

Comment:
I still believe the node version should be a separate column/row in the table

Reply:
How linked are the node versions and fabric versions?

Reply:
Just don't want it to look like we support node 8 with v2 if v2 doesn;'t support it

Reply:
I've updated the table - @awjh-ibm @heatherlp @lesleyannjordan @davidkel ... do you want to see if that answers the question :-)